src/core/meshcop/joiner.cpp

```diff
@@ -382,16 +382,16 @@ Error Joiner::Connect(JoinerRouter &aRouter)
     return error;
 }
 
-void Joiner::HandleSecureCoapClientConnect(bool aConnected, void *aContext)
+void Joiner::HandleSecureCoapClientConnect(MeshCoP::SecureTransport::ConnectEvent aEvent, void *aContext)
```

Comment:
not necessary to have `MeshCoP::` prefix here as Joiner class is in the same MeshCoP namespace as SecureTransport? same for Commissioner class as well.